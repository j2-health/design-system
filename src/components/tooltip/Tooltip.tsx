import { Tooltip as AntdTooltip, TooltipProps } from 'antd'

export type Props = Expand<TooltipProps>

const Tooltip = ({ children, placement = 'top', ...props }: Props) => {
  return (
    <AntdTooltip {...props} placement={placement}>
      {children}
    </AntdTooltip>
  )
}

export { Tooltip }
