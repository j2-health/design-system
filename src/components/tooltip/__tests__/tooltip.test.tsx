import { render } from '@testing-library/react'
import { Tooltip } from '../Tooltip'

describe('Tooltip', () => {
  it('should render correctly', () => {
    const { container } = render(
      <Tooltip title="The best healthcare">Aetna</Tooltip>
    )
    expect(container).toMatchSnapshot()
  })
})
