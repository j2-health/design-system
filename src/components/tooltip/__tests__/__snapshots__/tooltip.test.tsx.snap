// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tooltip should render correctly 1`] = `
<div>
  <div
    aria-describedby="test-id"
  >
    Aetna
  </div>
</div>
`;
